netcore/sample/HelloWorld/Program.cs

```diff
@@ -11,6 +11,8 @@ static void Main(string[] args)
             Console.WriteLine(typeof(object).Assembly.FullName);
             Console.WriteLine(System.Reflection.Assembly.GetEntryAssembly ());
             Console.WriteLine(System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription);
+
+	    GCMemoryInfo memInfo = GetGCMemoryInfo();
```

Comment:
There should be corefx unit tests to enable instead

Reply:
I hadn't intended to include this change in the PR; removing it.